crates/bevy-inspector-egui/src/inspector_egui_impls/bevy_impls.rs

```diff
@@ -57,7 +57,7 @@ pub fn entity_ui(
                     if options.despawnable && world.contains_entity(entity) {
                         if let Some(queue) = queue {
                             if egui_utils::label_button(ui, "✖ Despawn", egui::Color32::RED) {
-                                queue.push(bevy_ecs::system::Despawn { entity });
+                                queue.push(bevy_hierarchy::DespawnRecursive { entity });
```

Comment:
The migration guide suggests using a closure here

https://bevyengine.org/learn/migration-guides/0-12-to-0-13/#restore-support-for-running-fn-entitycommands-on-entities-that-might-be-despawned

```rust
move |world: &mut World| {
  world.entity_mut(entity).despawn_recursive();
}
```